frontend/src/components/RightSide/RightSide.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import { Aside, AsideNav } from './RightSide.styled';
+
+const RightAside = () => {
```

Comment:
```suggestion
const RightSidebar = () => {
```

sidebar !== aside

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/aside

![image](https://user-images.githubusercontent.com/41823689/149632824-208e6100-3009-4d0d-9c48-9cad9883c0c4.png)


![image](https://user-images.githubusercontent.com/41823689/149632818-5e5a542d-8297-498d-b483-cfb6b160d868.png)